src/pkg/registry/interceptor/interceptor.go

```diff
@@ -15,7 +15,6 @@
 package interceptor
 
 import "net/http"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380